### ISSUE TITLE
2.2 Conformance Model, Measurable Guidance - Plain Language suggestion

### ISSUE THREAD

**Issue by msellon** [w3c/silver#298]
Suggested edit to Measurable Guidance statement:

Certain accessibility guidance is quite clear and measurable. Others, far less so. Some needs of people with disabilities, especially cognitive and low vision disabilities, are not well served by guidance that can only be measured by pass/fail statements. Multiple means of measurement, in addition to pass/fail statements, allows for the inclusion of more accessibility guidance in WCAG 3.0.

===
I am also unsure what is intended with "allow for the inclusion of more accessibility guidance". Does this mean that WCAG 3.0 can contain more guidance? Or that the end product is more likely to be made accessible? If the latter, suggest rewording the last sentence to:

Allowing multiple means of measurement, rather than only pass/fail statements, encourages the creation of a more accessible end product.

===
Original text:
>Certain accessibility guidance is quite clear and measurable. Others, far less so. There are needs of people with disabilities, especially cognitive and low vision disabilities that are not well served by guidance that can only be measured by pass/fail statement. Multiple means of measurement, in addition to pass/fail statements, allow inclusion of more accessibility guidance.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by jspellman** [comment]
PR #619

**Comment by slauriat** [comment]
Thank you for comment! We've incorporated changes into pull request #619 to address.

**Comment by alastc** [comment]
Closed by #619
